springfox-bean-validators/src/main/java/springfox/bean/validators/plugins/Validators.java

```diff
@@ -119,12 +119,11 @@ public static boolean annotationMustBeApplied(ModelPropertyContext context, Clas
     }
```

Comment:
### First condition (or `Case #1`) - 114 line
If model does not have `@Valid` or `@Validated` annotation, the system should not enforce annotation constraints for this model. I think there must be some flag indicating how the system should do in this situation. Currently, the presence of annotations `@Valid` or `@Validated` does not affect the applicability of restrictions.